examples/navier-stokes/densitycurrent.h

```diff
@@ -217,86 +217,84 @@ static int DC(void *ctx, CeedInt Q,
   for (CeedInt i=0; i<Q; i++) {
     // Setup
     // -- Interp in
-    const CeedScalar rho     =   q[i+0*Q];
-    const CeedScalar u[3]    = { q[i+1*Q] / rho,
-                                 q[i+2*Q] / rho,
-                                 q[i+3*Q] / rho
-                               };
-    const CeedScalar E       =   q[i+4*Q];
+    const CeedScalar rho      =    q[i+0*Q];
+    const CeedScalar u[3]     =  { q[i+1*Q] / rho,
+                                   q[i+2*Q] / rho,
+                                   q[i+3*Q] / rho
+                                 };
+    const CeedScalar E        =    q[i+4*Q];
     // -- Grad in
-    const CeedScalar drho[3] = {  dq[i+(0+5*0)*Q],
-                                  dq[i+(0+5*1)*Q],
-                                  dq[i+(0+5*2)*Q]
-                               };
+    const CeedScalar drho[3]  =  { dq[i+(0+5*0)*Q],
+                                   dq[i+(0+5*1)*Q],
+                                   dq[i+(0+5*2)*Q]
+                                 };
     const CeedScalar du[3][3] = {{(dq[i+(1+5*0)*Q] - drho[0]*u[0]) / rho,
-                                  (dq[i+(1+5*1)*Q] - drho[1]*u[0]) / rho,
-                                  (dq[i+(1+5*2)*Q] - drho[2]*u[0]) / rho},
-                                 {(dq[i+(2+5*0)*Q] - drho[0]*u[1]) / rho,
+                                  (dq[i+(1+5*1)*Q] - drho[0]*u[1]) / rho,
+                                  (dq[i+(1+5*2)*Q] - drho[0]*u[2]) / rho},
```

Comment:
What is going on here? As I understand it, these three should come from the identity

    grad(rho u) = grad(rho) u + rho grad(u)

and the next three are

    grad(rho v) = grad(rho) v + rho grad(v)

The change introduced in this commit would mix `v` and `w` into the expression involving `rho u`.